modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -0,0 +1,72 @@
+= Multi-factor Authentication
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Multi-factor authentication provides an additional layer of verification for individual Anypoint Platform users.
+You can require all users in your organization to use multi-factor authentication and exempt specific accounts, or users can manually enable multi-factor authentication in their profile settings.
+
+When multi-factor authentication is enabled, users must verify their Anypoint Platform credentials using one of the following authentication methods:
```

Comment:
I think we enable users to choose more than one verifier for their account, right @maxwellgerber ?

```suggestion
When multi-factor authentication is enabled, users must verify their Anypoint Platform credentials using at least one of the following authentication methods:
```

Reply:
That's correct